src/Mod/Import/gltfGlbExt.py

```diff
@@ -0,0 +1,371 @@
+        pass
+
+    # Inject VarSet data into nodes
+    try:
+        if (
+            "nodes" in gltf_data
+            and isinstance(gltf_data["nodes"], list)
+            and "collected_varset_data" in globals()
+            and collected_varset_data
+        ):
+
+            for node in gltf_data["nodes"]:
+                if (
+                    isinstance(node, dict)
+                    and "name" in node
+                    and isinstance(node["name"], str)
+                    and node["name"] in collected_varset_data
+                ):
+
+                    if "extras" not in node:
+                        node["extras"] = {}
+                    elif not isinstance(node["extras"], dict):
+                        node["extras"] = {}
+
+                    node["extras"].update(collected_varset_data[node["name"]])
+    except (KeyError, TypeError, IndexError):
+        pass
+
+    return gltf_data
+
+
+def extract_json_from_glb(filename):
+    """
+    Extract JSON chunk from GLB file
+    """
+
+    try:
+        with open(filename, "rb") as f:
+            # Read GLB header
+            magic = f.read(4)
+            if magic != b"glTF":
+                return None
+
+            # Read JSON chunk header
+            json_chunk_length = struct.unpack("<I", f.read(4))[0]
+            json_chunk_type = f.read(4)
+
+            if json_chunk_type != b"JSON":
+                return None
+
+            json_data = f.read(json_chunk_length)
+            return json.loads(json_data.decode("utf-8"))
+
+    except (IOError, OSError, struct.error, json.JSONDecodeError, UnicodeDecodeError):
+        return None
+
+
+def write_json_to_glb(filename, gltf_data):
+    """
+    Write modified JSON back to GLB file
+    """
+
+    try:
+        with open(filename, "rb") as f:
+            original_data = f.read()
+
+        # Extract binary data (everything after JSON chunk)
+        json_str = json.dumps(gltf_data, separators=(",", ":")).encode("utf-8")
+
+        json_padding = (4 - (len(json_str) % 4)) % 4
+        json_str += b" " * json_padding
+
+        header_size = 12  # GLB header
+        json_chunk_header_size = 8  # JSON chunk header
+
+        original_json_length = struct.unpack("<I", original_data[12:16])[0]
+
+        binary_start = header_size + json_chunk_header_size + original_json_length
+
+        binary_data = original_data[binary_start:]
+
+        new_total_length = header_size + json_chunk_header_size + len(json_str) + len(binary_data)
+
+        with open(filename, "wb") as f:
+            f.write(b"glTF")
+            f.write(struct.pack("<I", 2))
+            f.write(struct.pack("<I", new_total_length))
+
+            f.write(struct.pack("<I", len(json_str)))
+            f.write(b"JSON")
+
+            f.write(json_str)
+
+            if binary_data:
+                f.write(binary_data)
+
+    except (IOError, OSError, struct.error, json.JSONEncodeError, UnicodeEncodeError):
```

Comment:
There is no class called `json.JSONEncodeError` -- you should be able to just drop this exception type from the list.